packages/application-shell/src/components/application-shell-provider/utils.spec.js

```diff
@@ -0,0 +1,29 @@
+import { getBrowserLocale } from './utils';
+
+describe('getBrowserLocale', () => {
+  let testWindow;
+  describe('when the locale is supported by the MC', () => {
+    beforeEach(() => {
+      testWindow = {
+        navigator: {
+          language: 'de-DE',
+        },
+      };
+    });
+    it('should return the locale', () => {
+      expect(getBrowserLocale(testWindow)).toBe('de-DE');
```

Comment:
Would it be easier to avoid beforeEach and just do

```js
expect(getBrowserLocale({
  navigator: {
     language: 'de-DE',
  }
})).toBe('de-DE');
```

maybe?